tests/compilation_tests/robot_test/traverse_standard_test.py

```diff
@@ -26,7 +26,19 @@ def test1(self):
         allowed_dist_error = 0.05
         move_to_target_node(r2d2_state,
                             (42.444250, -66.483682), allowed_dist_error, database)
+    
+    #Runs test1 using simple_move_to_target_node
+    def test1_simple(self):
+        r2d2_state = Robot_State(xpos=42.444250, ypos=-76.483682, heading=0,
+                                 epsilon=0.2, max_velocity=0.5, radius=0.2, phase=2, position_kp=.1, heading_kp=.1, position_kd=0.0, heading_kd=0.0
```

Comment:
iirc, most of these params have default args. Do we need to specify some of these args for testing purposes?

Reply:
i also see this defined in test1. i think it would be better to initialize these r2d2 within the class itself to remove duplicate code

Reply:
Made class variables and removed duplicate code.